code/src/index.css

```diff
@@ -1,13 +1,197 @@
+@import url('https://fonts.googleapis.com/css2?family=Shadows+Into+Light&display=swap');
+
+:root {
+  font-size: 19px;
+}
+
+*, *:after, *:before {box-sizing: border-box;}
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: 'Shadows Into Light', cursive;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-image: url(../public/bree-anne-XnpWqPZNfXE-unsplash.jpg);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100vw;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.paper {
+  width: 98vw;
+  height: auto;
+  min-height: 80vh;
+  background-color: #fff;
+  box-shadow: 3px 2px 2px 0px rgba(0,0,0,0.4);
+  position: relative;
+  padding: 40px 0 40px 0;
+  margin: 20px 0 10px 0;
+}
+.paper::before {
+  content: '';
+  width: 2px;
+  height: 100%;
+  background-color: rgba(255, 0, 0, 0.6);
+  position: absolute;
+  top: 0;
+  left: 40px;
+}
+.pattern {
+  min-height: 80vh;
+  height: auto;
+  background-image: repeating-linear-gradient(white 0px, white 24px, teal 25px);
+}
```

Comment:
It is so impressive you created the note paper in css! It's inspiring! Great work!